cgmanifest.json

```diff
@@ -64,32 +64,35 @@
     {
       "Component": {
         "Type": "linux",
-        "Other": {
+        "linux": {
           "Name": "gcovr",
           "Version": "3.4-1",
-          "DownloadUrl": "https://ubuntu.pkgs.org/18.04/ubuntu-universe-amd64/gcovr_3.4-1_all.deb.html"
+          "Distribution": "Ubuntu",
+          "Release": "18.04"
```

Comment:
I think we may also need Pool-URL and/or Key-URL.

Maybe follow up with cg folks or look up other repos that use the `linux` component type, to see what is the full amount of detail we need to provide.

> Valid values for distribution: Debian, Alpine, Rhel, Centos, Fedora, Ubuntu. Pool-URL Example (http://archive.ubuntu.com/ubuntu) and Key-URL Example (http://archive.ubuntu.com/ubuntu/project/ubuntu-archive-keyring.gpg)

Reply:
Yes this linux dependency looks correct

Reply:
Where do we typically find "valid" values for Pool and Key URLs for the different distros?